cmdstanpy/cmdstan_args.py

```diff
@@ -336,7 +336,7 @@ def validate(self, chains=None) -> None:  # pylint: disable=unused-argument
                     'init_alpha must not be set when algorithm is Newton'
                 )
             if isinstance(self.init_alpha, Real):
-                if self.init_alpha < 0:
+                if self.init_alpha <= 0:
```

Comment:
must be greater than 0 - https://github.com/stan-dev/cmdstan/blob/3e93d44ca33200bb3f09218451ffa36ab3c8a01d/src/cmdstan/arguments/arg_init_alpha.hpp#L13

Reply:
my mistake - this change it correct - got things backwards.
would `if not self.init_alpha > 0` be easier to understand?